player/js/utils/imagePreloader.js

```diff
@@ -1,14 +1,14 @@
 var ImagePreloader = (function(){
 
-    var proxyImage = (function(){
+    function createProxyImage() {
```

Comment:
@grhbit By doing this change, every failed image would be creating a new canvas instance.
What is the intent behind this?

Reply:
this way lgtm，cc @bodymovin 
// imagePreloader.js
function getProxyImage() {
  var proxyImage;
  return  function() {
    if (proxyImage) {
      return proxyImage;
    }
    proxyImage = createProxyImage();
    return proxyImage;
  }
}

// in function imageLoaded
ob.img = getProxyImage